project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -137,4 +139,8 @@ PlaceDetails getPlaceDetails(PlaceDetailsRequest request)
             throws ApiException, InterruptedException, IOException {
         return request.await();
     }
+
+    private static String createCuisenesQuery() {
+        return CUISINES.stream().collect(joining("|"));
```

Comment:
Test?

Reply:
By the way, I think this can be simplified by calling
`String.join("|", CUISINES`);`

I don't see the benefit of using streams here.

Reply:
Yes the join works, thanks.

Regarding the testing, isn't it enough that it is tested as part of the fetch testing? Since CUISINES is hardcoded in this version we only need it to work for this specific list of cuisines without any corner cases, and so this is tested in the current tests as part of fetch testing.

Reply:
The problem is that `getCuisinesQuery` is not tested now, and you'll probably not test it in the next PR (as it's an existing functionality). So the recommended way is to add the test directly when you add the functionality.